### PR TITLE
[ACR] Add logtemplate and systemtask changes for ACR Tasks

### PR DESCRIPTION
**Pull request by savaradh**
* Add log template changes to task, run, build

* Add systemtask changes to task

**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by savaradh** [comment]
cc: @northtyphoon

**Comment by yonzhan** [comment]
ACR

File: src/azure-cli/azure/cli/command_modules/acr/task.py [hunk 1]
```diff
@@ -72,11 +73,34 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
...
+        try:
+            return client.create(resource_group_name=resource_group_name,
+                                 registry_name=registry_name,
+                                 task_name=task_name,
+                                 task_create_parameters=task_create_parameters)
+        except ValidationError as e:
+            raise CLIError(e)
+
+    if not context_path:
+        raise CLIError("If the task is not a System Task, --context-path must be provided.")
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/acr/task.py:102]: can this message be put to help message?

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@ azure-mgmt-cognitiveservices==6.2.0
 azure-mgmt-compute==13.1.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
-azure-mgmt-containerregistry==3.0.0rc14
+azure-mgmt-containerregistry==3.0.0rc15
```
Comment (yungezz) [inline src/azure-cli/requirements.py3.Darwin.txt:33]: pls make sure you run live test to regress all ACR tests.

Comment (fengzhou-msft) [reply src/azure-cli/requirements.py3.Darwin.txt:33]: When will azure-mgmt-containerregistry 3.0.0 be released officially? We recently got some requirements from Homebrew to only use stable versions of the dependencies.

File: src/azure-cli/azure/cli/command_modules/acr/task.py (hunk 1 above)
Comment (savaradh) [reply src/azure-cli/azure/cli/command_modules/acr/task.py:102]: @yungezz updated the help section.

File: src/azure-cli/requirements.py3.Darwin.txt (hunk 2 above)
Comment (yugangw-msft) [reply src/azure-cli/requirements.py3.Darwin.txt:33]: @fengzhou-msft,  azure-mgmt-containerregistry is a multi-api package including both ga and prewiew api versions. We can release a new non preview/rc package in the near future and submit a new PR. Does this sound reasonable?
Meanwhile, could you please share the context for why Homebrew would only allow stable versions?

Comment (yungezz) [reply src/azure-cli/requirements.py3.Darwin.txt:33]: HI @yugangw-msft pls see the conversation https://github.com/Homebrew/homebrew-core/pull/61520#discussion_r492907434. We do let them know rc sdk version might contain both stable and preview API, and CLI has preview commands by https://github.com/Homebrew/homebrew-core/pull/61520#issuecomment-698424333. while from our end, we also want to let people aware to try to use stable version as possible.

**Review by fengzhou-msft — APPROVED** [review]

**Comment by savaradh** [comment]
@yugangw-msft Can you please review this PR and approve it? It has log template changes for Tasks

**Review by yugangw-msft — APPROVED** [review]

**Comment by savaradh** [comment]
@fengzhou-msft can you please merge this request?

**Pull request merged**